src/SamplesApp/SamplesApp.Skia.Gtk/SamplesApp.Skia.Gtk.csproj

```diff
@@ -24,11 +24,6 @@
 		<OutputType>Exe</OutputType>
 	</PropertyGroup>
 
-	<PropertyGroup>
-		<!-- Workaround for https://github.com/unoplatform/uno/discussions/5007 -->
-		<SynthesizeLinkMetadata>true</SynthesizeLinkMetadata>
```

Comment:
There seems to be three other instances of this workaround :
1. src/SamplesApp/SamplesApp.netcoremobile/SamplesApp.netcoremobile.csproj#L41
2. src/SolutionTemplate/UnoSolutionTemplate/UWP/UnoQuickStart.Uwp.csproj#L20
3. src/SolutionTemplate/UnoSolutionTemplate.netcore/UWP/UnoQuickStart.Uwp.csproj#L20